subsys/bluetooth/services/Kconfig.dis

```diff
@@ -85,4 +99,47 @@ config BT_GATT_DIS_PNP_VER
 
 endif #BT_GATT_DIS_PNP
 
+config BT_GATT_DIS_SERIAL_NUMBER
+	bool "Enable DIS Serial number characteristic"
+	help
+	 Enable Serial Number characteristic in Device Information Service.
+
+config BT_GATT_DIS_SERIAL_NUMBER_STR
+	string "Serial Number"
+	depends on BT_GATT_DIS_SERIAL_NUMBER
+	help
+	 Enable Serial Number characteristic in Device Information Service.
+
+config BT_GATT_DIS_FIRMWARE_REVISION
```

Comment:
Can we make those shorter by using `FW`, `HW` and `SW` instead of `FIRMWARE`, `HARDWARE` and `SOFTWARE`?

Reply:
Can do that